src/test/java/com/waverleysoftware/LoginToApplication.java

```diff
@@ -0,0 +1,32 @@
+package com.waverleysoftware;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+public class LoginToApplication extends BaseClass {
+    WebElement username = driver.findElement(By.id("email"));
+    WebElement password = driver.findElement(By.id("pass"));
+    WebElement Login = driver.findElement(By.id("u_0_2"));
+    String pageTitle = driver.getTitle();
+
+    @Test(description = "Perform login to application")
+    public void loginToApplication() throws Throwable {
+
+        username.sendKeys("artembashlak@gmail.com");
+        password.sendKeys("fuck123.");
+        Login.click();
+        Thread.sleep(200);
+        System.out.println(pageTitle);
+    }
+
+
+    @Test(description = "Perform login with invalid credentials")
+    public void failLoginToApplication() {
```

Comment:
you're using the same steps for different tests... seems like a duplication...
anyway, you need to revise your tests structure, as technically they're not testing anything...